OrbitLinuxTracing/TracerThread.h

```diff
@@ -16,6 +17,7 @@
 #include <regex>
 #include <vector>
 
+#include "../../OrbitGl/TracepointCustom.h"
```

Comment:
This looks very wrong. We do not want a dependency from the linux specific tracing code into the ui code in OrbitGl (this is why you have to specify the path in this weird way, the includes are not (really) visible). 
Can you move TracepointCustom.h to a place that is shared between the service and the UI? OrbitCore maybe?

Reply:
Of course. Thanks for the advice!